core/src/layers/complete.rs

```diff
@@ -174,7 +174,12 @@ impl<A: Access> CompleteAccessor<A> {
         }
 
         if path == "/" {
-            return Ok(RpStat::new(Metadata::new(EntryMode::DIR)));
+            let meta = if capability.stat {
+                self.inner.stat(path, args).await?.into_metadata()
```

Comment:
Hi, it's intended that we don't send `stat` for `/`. The service should always make sure `/` exists.